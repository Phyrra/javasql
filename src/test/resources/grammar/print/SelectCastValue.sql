select cast(f as int) as i