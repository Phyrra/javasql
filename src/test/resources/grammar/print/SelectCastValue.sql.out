sql
    statement
        select
            value
                aliased
                    primary
                        cast
                            primary
                                field
                                    f
                            data type: int
                    i